.travis/setup-kubernetes.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl
+
+mkdir -p /tmp/openshift
+wget https://github.com/openshift/origin/releases/download/v3.7.0/openshift-origin-client-tools-v3.7.0-7ed6862-linux-64bit.tar.gz -O openshift.tar.gz
+tar xzf openshift.tar.gz -C /tmp/openshift --strip-components 1  
+sudo cp /tmp/openshift/oc /usr/bin
```

Comment:
If on Travis CI we can't use `oc` tool due to old systemd, why we are pulling the tool here ?

Reply:
I just copied this script from enmasse, since @lulf has done all the hard work of making this work. I think openshift is downloaded because (until he gave up due to systemd issue) he was also trying to make openshift work too. If I remove it here we end up diverging from enmasse. Let's see what @lulf says about fixing this in enmasse.

Reply:
@lulf added `oc` download specifically in [this commit](https://github.com/EnMasseProject/enmasse/commit/04fc947aa68fa08b2da6f64f6d368e2c906df06c), but I don't know what "useful debug info" he needed it for.

Reply:
Apparently they use `oc` for things like `rsh` and `extract`. With `rsh` we could test some interesting corner cases like killing processes etc, so I think we should keep the `oc` download for now. We can always remove it later when it turns out that it's making the ci builds too slow.